package.json

```diff
@@ -37,7 +37,7 @@
     "jscoverage": "^0.6.0",
     "mocha": "^2.3.4",
     "mocha-lcov-reporter": "^1.0.0",
-    "protagonist": "",
```

Comment:
You should probably pin to `^1.0.0` since future breaking changes will break your tests.